specs/mount.go

```diff
@@ -0,0 +1,92 @@
+package specs
+
+// This file was generated by the swagger tool.
+// Editing this file might prove futile when you re-run the swagger generate command
+
+import (
+	strfmt "github.com/go-openapi/strfmt"
+
+	"github.com/go-openapi/errors"
+	"github.com/go-openapi/validate"
+)
+
+/*Mount mount
+
+swagger:model Mount
+*/
+type Mount struct {
+
+	/* destination
```

Comment:
What's missing for it to be compliant with the style guide. The goals are that the generated code passes golint etc checks.